ui/src/views/Documentation/Reference/Entry.jsx

```diff
@@ -105,6 +105,8 @@ export default class Entry extends Component {
       this.props.entry.name === window.location.hash.slice(1) ||
       this.props.entry.type === window.location.hash.slice(1) ||
       encodeURIComponent(path(['content', '$id'], this.props.entry)) ===
+        window.location.hash.slice(1) ||
+      encodeURIComponent(this.props.entry.schema.$id) ===
```

Comment:
I think we can remove this extra logic and just replace `content` with `schema`.

Reply:
@helfi92 Thankyou for helping out here. This is my first open source contribution :) 
I have made the changes as requested.